### PR TITLE
added co-author map to author profile

### LINKED ISSUE #1156: On author profile, add map of institutions of collaborators
similar to 
"Map of organizations associated with works about the topic"
https://tools.wmflabs.org/scholia/topic/Q202864#Map-of-organizations

### PR DESCRIPTION
**Pull request by Daniel-Mietchen**
fixes #1156

### CODE REVIEW

File: scholia/app/templates/author.html [hunk 1]
```diff
@@ -500,13 +506,6 @@ <h2 id="Academic tree">Academic tree</h2>
 </div>
 
 
-<h2 id="Locations">Locations</h2>
```
Comment (egonw) [inline scholia/app/templates/author.html:503]: @Daniel-Mietchen, is this section intentionally removed? I cannot make up from the PR or the bug report if it is meant to replace this section.

**Review by egonw — CHANGES_REQUESTED** [review]

**Comment by egonw** [comment]
`tox -e flake8` passed.

File: scholia/app/templates/author.html [hunk 2]
```diff
@@ -45,7 +45,7 @@
     ?venue
     (GROUP_CONCAT(DISTINCT ?topic; separator=" // ") AS ?topics)
   WHERE {
-	  ?work wdt:P50 wd:{{ q }} .
+      ?work wdt:P50 wd:{{ q }} .
```
Comment (fnielsen) [inline scholia/app/templates/author.html:48]: There is an indentation change

File: scholia/app/templates/author.html (hunk 1 above)
Comment (Daniel-Mietchen) [reply scholia/app/templates/author.html:503]: I have now put it back in as "other locations", after the co-author map.

File: scholia/app/templates/author.html (hunk 2 above)
Comment (Daniel-Mietchen) [reply scholia/app/templates/author.html]: Fixed.

**Comment by Daniel-Mietchen** [comment]
@egonw @fnielsen I made changes in response to your comments. Please take another look.

Overview:

Screenshot_2020-10-05 Scholia

**Pull request merged**